Documents/AnalyticsToolDesignDocument.md

```diff
@@ -0,0 +1,48 @@
+# VFS For Git: Analytics Tooling Design Document
+
+This document details the design of and plan for the **VFS4G analytics tool**. The tool aims to help users identify reasons why their enlistment may be slowing down and the `gvfs` commands running slowly. By collecting and reporting statistics, the power to uncover and hopefully fix problems as they arrive moves to the user, empowering them to use the software to its fullest potential.

```

Comment:
> The tool aims to help users identify reasons why their enlistment may be slowing down and the `gvfs` commands running slowly.

- To confirm, do you mean `git` or `gvfs` commands here?
- Will the tool help identify which commands are running slowly, or why they might be running slowly?

Reply:
I do mean git, thanks for catching that. I think tracking command run times might be a useful metric to capture so I'll go ahead and add it to the list of statistics planned for reporting